core-web/nx.json

```diff
@@ -1,132 +1,138 @@
+      "changeDetection": "OnPush"
     },
-    "defaultProject": "dotcms-ui",
-    "generators": {
-        "@nx/angular:application": {
-            "style": "scss",
-            "linter": "eslint",
-            "unitTestRunner": "jest",
-            "e2eTestRunner": "cypress"
-        },
-        "@nx/angular:library": {
-            "linter": "eslint",
-            "unitTestRunner": "jest"
-        },
-        "@nx/angular:component": {
-            "style": "scss",
-            "changeDetection": "OnPush"
-        },
-        "@nx/angular": {
-            "application": {
-                "linter": "eslint"
-            },
-            "library": {
-                "linter": "eslint"
-            },
-            "storybook-configuration": {
-                "linter": "eslint"
-            }
-        },
-        "@nrwl/angular:application": {
-            "style": "scss",
-            "linter": "eslint",
-            "unitTestRunner": "karma",
-            "e2eTestRunner": "cypress"
-        },
-        "@nrwl/angular:library": {
-            "linter": "eslint",
-            "unitTestRunner": "karma"
-        },
-        "@nrwl/angular:component": {
-            "style": "scss",
-            "changeDetection": "OnPush"
-        },
-        "@nrwl/react": {
-            "application": {
-                "babel": true
-            },
-            "library": {
-                "unitTestRunner": "jest"
-            }
-        }
+    "@nx/angular": {
+      "application": {
+        "linter": "eslint"
+      },
+      "library": {
+        "linter": "eslint"
+      },
+      "storybook-configuration": {
+        "linter": "eslint"
+      }
     },
-    "$schema": "./node_modules/nx/schemas/nx-schema.json",
-    "targetDefaults": {
-        "build": {
-            "dependsOn": ["^build"],
-            "inputs": ["production", "^production"]
-        },
-        "test": {
-            "inputs": ["default", "^production", "{workspaceRoot}/karma.conf.js"]
-        },
-        "e2e": {
-            "inputs": ["default", "^production"]
-        },
-        "lint": {
-            "inputs": ["default", "{workspaceRoot}/.eslintrc.json"]
-        },
-        "build-storybook": {
-            "inputs": [
-                "default",
-                "^production",
-                "{workspaceRoot}/.storybook/**/*",
-                "{projectRoot}/.storybook/**/*",
-                "{projectRoot}/tsconfig.storybook.json"
-            ]
-        },
-        "@nx/jest:jest": {
-            "inputs": ["default", "^production", "{workspaceRoot}/karma.conf.js"],
-            "cache": true,
-            "options": {
-                "passWithNoTests": true
-            },
-            "configurations": {
-                "ci": {
-                    "ci": true,
-                    "codeCoverage": true
-                }
-            }
-        }
+    "@nrwl/angular:application": {
+      "style": "scss",
+      "linter": "eslint",
+      "unitTestRunner": "karma",
```

Comment:
should this be jest?

Reply:
I'm not sure what is `@nrwl` thought, maybe we dont need to change this.

Reply:
That section of that file, is for set the defaults when you create a new (in this case) application